gustaf/create/faces.py

```diff
@@ -6,30 +6,121 @@
 
 from gustaf.faces import Faces
 from gustaf import utils, create
+from gustaf import settings
 
 
-def box(bounds=[[0, 0], [1, 1]], resolutions=[2, 2]):
-    """
-    Create structured quadrangle block mesh.
+def box(
+        bounds=[[0, 0], [1, 1]],
+        resolutions=[2, 2],
+        simplex=False,
+        backslash=False
+):
+    """Create structured quadrangle or triangle block mesh.
+
     Parameters
     -----------
     bounds: (2, 2) array
         Minimum and maximum coordinates.
     resolutions: (2) array
         Vertex count in each dimension.
+    simplex: boolean
+        If true, Mesh will be triangular (simplex).
+
     Returns
     --------
     face_mesh: Volumes
     """
+
     if np.array(bounds).shape != (2, 2):
         raise ValueError("Bounds must have a dimension of (2, 2).")
     if len(resolutions) != 2:
         raise ValueError("Resolutions must have two entries.")
     if not np.greater(resolutions, 1).all():
         raise ValueError("All resolution values must be at least 2.")
 
-    vertex_mesh = create.vertices.raster(bounds, resolutions)
-    connectivity = utils.connec.make_quad_faces(resolutions)
-    face_mesh = Faces(vertex_mesh.vertices, connectivity)
+    if simplex:
+        # create quad mesh as basis
+        quad_mesh = box(bounds=bounds, resolutions=resolutions)
+
+        # turn into triangles
+        face_mesh = tosimplex(quad_mesh, backslash)
+
+    else:
+        vertex_mesh = create.vertices.raster(bounds, resolutions)
+        connectivity = utils.connec.make_quad_faces(resolutions)
+        face_mesh = Faces(vertex_mesh.vertices, connectivity)
 
     return face_mesh
+
+
+def tosimplex(quad, backslash=False):
```

Comment:
rename function to `quad_to_simplex()` or `to_simplex()`?

Reply:
I suggested `tosimplex` since we have `tofaces`, `tovertices` and so on

Reply:
so we keep ```tosimplex``` ?

Reply:
sure. @clemens-fricke are you okay with this?

Reply:
Thanks. Works for me.